web/controllers/notes_controller.ex

```diff
@@ -2,6 +2,13 @@ defmodule Notox.NoteController do
   use Notox.Web, :controller
   alias Notox.Note
 
+  def index(conn, %{"search" => %{"query" => query}}) do
+    notes =
+      Note.search(Note, query)
```

Comment:
Cool. I like that you pass in the query instead of hardcoding it into the function body. Only suggestion is to make `Note` part of the pipeline.